acceptance/bundle/deployment/bind/experiment/test.toml

```diff
@@ -1,6 +1,32 @@
-Local = false
+Local = true
 Cloud = true
 
 [[Repls]]
 Old = "[0-9]{3,}"
-New = "[NUMID]"
\ No newline at end of file
+New = "[NUMID]"
+
+[[Server]]
+Pattern = "POST /api/2.0/mlflow/experiments/create"
+Response.Body = '''
+{
+    "experiment_id":"1234567890"
+}
+'''
+
+[[Server]]
+Pattern = "GET /api/2.0/mlflow/experiments/get"
+Response.Body = '''
+{
+    "experiment_id": "1234567890",
+    "experiment": {
+        "name": "/Users/[USERNAME]/test-experiment6260d50f-e8ff-4905-8f28-812345678903",
+        "lifecycle_stage": "active"
+    }
+}
+'''
+
+[[Server]]
+Pattern = "POST /api/2.0/mlflow/experiments/update"
+
+[[Server]]
+Pattern = "POST /api/2.0/mlflow/experiments/delete"
```

Comment:
Nit: not having a trailing newline likely triggers unintended diffs in future edits.

If you're using VS Code then the settings in this repository should already fix this on save ([ref](https://github.com/databricks/cli/blob/e2339192120e648a3ada28c6a73df67acd0b1fb8/.vscode/settings.json#L14-L15)).

Reply:
noted, and changed my ide settings to comply in future commits.
also something that a linter could catch?